parent/pom.xml

```diff
@@ -18,7 +18,7 @@
   </description>
 
   <properties>
-    <version.quarkus>2.8.2.Final</version.quarkus>
+    <version.quarkus>2.16.3.Final</version.quarkus>
```

Comment:
🙃 Quarkus [just published `2.16.4.Final`](https://central.sonatype.com/artifact/io.quarkus/quarkus-core/2.16.4.Final) three days ago, do we want to bump to that? We don't have to if that involves too much follow-up stuff in my view. But if it's low effort, maybe it's worth being on the latest patch level. 🙂

Reply:
I don't see show stopper so I bump to 2.16.4 version. Requesting a new review.